ontology/wot-security.ttl

```diff
@@ -84,7 +84,7 @@
                                              
                                              rdfs:label "identity" ;
                                              
-                                             rdfs:comment "Identifier validating access by providing information unique to the resource." .
+                                             rdfs:comment "Identifier providing information unique to the resource which can be used for selection or confirmation." .
```

Comment:
Not unique to the resource!
> Unique name within the security configuration referring to a particular secret for gaining access.

Reply:
Ok, I will delete the phrase "unique to the resource".   So it will be "Identifier providing information which..."